task3/script.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById('ul');
+
+for (let i =1; i <=2; i++) {
```

Comment:
[IMO]

> for (let i =1; i <=2; i++) {

It is recommended that you assign the number "2" to a constant so that you know what number is being used in the for statement.

Reply:
Thank you for the review.

I understand.
It became easier to understand by assigning it to a constant.

Reply:
We have confirmed that it is defined as a constant!